### ISSUE TITLE
Full cover width seems not work under Windows

### ISSUE THREAD

**Issue by parkmino** [CDrummond/cantata#1252]
as attached.
![full-width-cover](https://user-images.githubusercontent.com/12131158/39337415-c638f256-49f7-11e8-8bde-1e6c73aa130f.png)

**Comment by CDrummond** [comment]
Yeah, I can confirm. Odd - is probably due to the Qt version used. However, Windows is not a priority for me (never use it), and this is a minor feature - so I'll probably just remove it for Windows builds.
